docs/PRIVACY.md

```diff
@@ -0,0 +1,46 @@
+
+Our Apps are not intended to be used by or to solicit information of any kind from children under age 13.
+
+## How We Use Data
+
+The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see third party tools). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
+
+## Disclaimer
+
+THE MATERIALS EMBODIED IN THIS SERVICE ARE PROVIDED “AS-IS” AND WITHOUT WARRANTY OF ANY KIND, EXPRESSED, IMPLIED OR OTHERWISE, INCLUDING WITHOUT LIMITATION, ANY WARRANTY OF FITNESS FOR A PARTICULAR PURPOSE. IN NO EVENT SHALL THE WORLD HEALTH ORGANIZATION BE LIABLE TO YOU OR ANYONE ELSE FOR ANY DIRECT, SPECIAL, INCIDENTAL, INDIRECT OR CONSEQUENTIAL DAMAGES OF ANY KIND, OR ANY DAMAGES WHATSOEVER, WHETHER OR NOT ADVISED OF THE POSSIBILITY OF SUCH LOSS, HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE POSSESSION, USE OR PERFORMANCE OF THIS SERVICE.
+
+## Data Retention and Transfer
+
+We retain and use Personal Data and Usage Data only as long as is necessary for the purposes set out in our Privacy Policy and this Privacy Notice, or to the extent necessary to comply with legal obligations, resolve disputes, or enforce legal agreements and policies. We transfer the data, including Personal Data, to the U.S.A. for processing and storage, and by using or submitting information via the Service you consent to such transfer.
+
+## Disclosure and Security of Data
+
+We may disclose your Personal Data in the good faith belief that such action is necessary to: comply with a legal obligation, protect and defend our rights or property, prevent or investigate possible wrongdoing in connection with the Service, protect the personal safety of users of the Service or the public, or protect against legal liability. We use commercially reasonable means to protect any Personal Data you provide to us.
+
+## Current Third-Party Tools
+
+Currently in connection with the WHO COVID-19 App we use the following third-party services and application stores. We may use additional services for testing and internal use.
+
+* Apple Store (iTunes Store) (Privacy Policy)
+* Google Analytics (Privacy Policy and Terms)
+* Google Cloud (Privacy Policy)
+* Google Firebase (Privacy Policy)
+* Google Play Store (Privacy Policy)
```

Comment:
```suggestion
* Apple Store (iTunes Store) ([Privacy Policy](https://www.apple.com/legal/privacy/en-ww/))
* Google Analytics ([Privacy Policy](https://support.google.com/analytics/answer/6004245) and [Terms](https://www.google.com/analytics/terms/))
* Google Cloud ([Privacy Policy](https://cloud.google.com/security/privacy))
* Google Firebase ([Privacy Policy](https://firebase.google.com/support/privacy))
* Google Play Store ([Privacy Policy](https://policies.google.com/privacy?hl=en&gl=us))
```

Reply:
Fixed. Will commit in just a minute when I fix the rest of this page's requests.